src/config/config.ts

```diff
@@ -0,0 +1,36 @@
+export const DOCUMENT_STORE_CREATOR_ROPSTEN = "0x4077534e82C97Be03A07FB10f5c853d2bC7161FB";
```

Comment:
I'm a bit lost with this file let's discuss about it tmr

Reply:
Yep sure